chart_deps/konghq/plugins/values.yaml

```diff
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 # Configur KongPlugins CRD
 
-plugins: []
+plugins: {}
```

Comment:
Hmm why are we changing this attr type here :thinking: ?

Reply:
return [ ]